### DISCUSSION TITLE
How to properly specify projectId in the CLI

### DISCUSSION THREAD

**Discussion by bilarallen** [currents-dev/cypress-currents-reporter#77] [Q&A] [answered]
Hello, may I ask how we can specify the `projectId` in the CLI when running a test with the cypress cloud? I have tried it before in the currents-cli like `npx currents run --parallel --record --key XXXXX --config projectId=XXXX--ci-build-id test-1`

But whenever I do the same for cypress cloud `npx cypress-cloud  --parallel --record --key XXXX --config projectId=XXXX --ci-build-id hello-cypress-cloud-test-2 `it doesn't work.
![image](https://user-images.githubusercontent.com/65072341/224537523-088ccb2b-bcc6-4bab-b79e-994219992e79.png)

I have also tried to specify it using `--env` but it also doesn't work
![image](https://user-images.githubusercontent.com/65072341/224538657-6d2a7c83-f600-4596-aa5a-fefe807f92ed.png)

One way I am thinking is to set it as an environment variable in a file but I don't know how to do this and set that env file in the CLI.

Context: In my single automation project I have to run 2 different npx commands and project ids and each has its own Jenkins file.

Hope someone can help. Thanks!

**Comment by agoldis** [comment, marked as answer]
@bilarallen you can use an environment variable `CURRENTS_PROJECT_ID`, or tweak the value in `currents.config.js` (see [example](https://github.com/currents-dev/cypress-cloud/blob/main/examples/webapp/currents.config.js#L8)). You can also create a feature request if those methods are not enough

**Reply by bilarallen** [reply]
Thanks @agoldis I was able to do it by adding env variable in Jenkins :)
